src/Utopia/Messaging/Adapters/SMS/Clickatell.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\SMS;
+
+use Utopia\Messaging\Adapters\SMS as SMSAdapter;
+use Utopia\Messaging\Messages\SMS;
+
+// Reference Material
+// https://docs.clickatell.com/channels/sms-channels/sms-api-reference/#tag/SMS-API/operation/sendMessageREST_1
+class Clickatell extends SMSAdapter
+{
+    /**
+     * @param  string  $apiKey Clickatell API Key
+     */
+    public function __construct(
+  private string $apiKey,
+ ) {
```

Comment:
Looks like the indenting is a little off in all the adapters, not sure why the linter isn't picking this up, please indent all the constructor parameters with 8 spaces :pray: